<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gauss Noise Filter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>180</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Noise %</string>
    </property>
   </widget>
   <widget class="QPushButton" name="genButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="noiseRateEdit">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>200</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="signalsTable">
    <property name="geometry">
     <rect>
      <x>656</x>
      <y>10</y>
      <width>361</width>
      <height>135</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>2</number>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>22</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>22</number>
    </attribute>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>x</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>y</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>sx</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>sy</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>A</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>64</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>64</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>100</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>150</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>150</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>100</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>100</string>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string>150</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="addRowButton">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>150</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delRowButton">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>150</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="clipRateEdit">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>200</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>90.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>180</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Clip %</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>625</width>
      <height>650</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="src">
     <attribute name="title">
      <string>Source</string>
     </attribute>
     <widget class="QGraphicsView" name="cleanView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>625</width>
        <height>625</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="noise">
     <attribute name="title">
      <string>Noise</string>
     </attribute>
     <widget class="QGraphicsView" name="noiseView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>625</width>
        <height>625</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="fft">
     <attribute name="title">
      <string>FFT</string>
     </attribute>
     <widget class="QGraphicsView" name="fftView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>625</width>
        <height>625</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="restored">
     <attribute name="title">
      <string>Restored</string>
     </attribute>
     <widget class="QGraphicsView" name="restoredView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>625</width>
        <height>625</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="sizeEdit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>200</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>512</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>180</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Noise</string>
    </property>
   </widget>
   <widget class="QPushButton" name="noiseButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Noise</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
